public/app/features/logs/components/log-context/LogRowContextModal.tsx

```diff
@@ -180,7 +180,14 @@ export const LogRowContextModal: React.FunctionComponent<LogRowContextModalProps
 
   const onChangeLimitOption = (option: SelectableValue<number>) => {
     setLoadMoreOption(option);
-    setLimit(option.value!);
+    if (!option.value) {
```

Comment:
I'm a bit confused by this line.

```suggestion
    if (option.value) {
```

Reply:
I am as well. Thanks for catching that!